### PR TITLE
Upgrade packages

### PR DESCRIPTION
**Pull request by revathskumar**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -112,35 +116,44 @@ gulp.task("sign", function() {
...
+  parallel(manifestVersion, manifestAppVersion, packageVersion),
+  changelog,
+  commitChanges,
+  pushChanges,
+  createNewTag,
+  parallel(xpi, zipDist, zipApp),
+  sign
+);
+
+// gulp.task("release:prod", function(done) {
```
Comment (aksh93) [inline gulpfile.js]: We can go ahead remove this commented out code?

File: gulpfile.js [hunk 2]
```diff
@@ -1,109 +1,113 @@
...
+      })
+  );
+}
 
-gulp.task('push-changes', function (done) {
-  git.push('origin', 'master', done);
-});
+function pushChanges(done) {
+  // git.push('origin', 'master', done);
+  console.log("push");
```
Comment (aksh93) [inline gulpfile.js]: Please revert the `console.log` here

File: gulpfile.js [hunk 3]
```diff
@@ -1,109 +1,113 @@
...
-gulp.task('create-new-tag', function (done) {
+function createNewTag(done) {
   var version = getPackageJsonVersion();
-  git.tag(version, version, function (error) {
+  git.tag(version, version, function(error) {
     if (error) {
       return done(error);
     }
-    git.push('origin', 'master', {args: '--tags'}, done);
+    console.log("push with tags");
```
Comment (aksh93) [inline gulpfile.js]: Here as well

Comment (revathskumar) [reply gulpfile.js]: Done

File: gulpfile.js (hunk 2 above)
Comment (revathskumar) [reply gulpfile.js]: Done.

**Pull request merged**